manuskript/functions/spellchecker.py

```diff
@@ -1,45 +1,111 @@
 
+import os, gzip, json
 from PyQt5.QtCore import QLocale
+from collections import OrderedDict
+from manuskript.functions import writablePath
 
 try:
     import enchant
 except ImportError:
     enchant = None
 
+try:
+    import spellchecker as pyspellchecker
+except ImportError:
+    pyspellchecker = None
+
+
 class Spellchecker:
     dictionaries = {}
+    # In order of priority
+    implementations = []
 
     def __init__(self):
         pass
 
     @staticmethod
     def isInstalled():
-        return enchant is not None
+        for impl in Spellchecker.implementations:
+            if impl.isInstalled():
+                return True
+        return False
+
+    @staticmethod
+    def supportedLibraries():
+        libs = []
+        for impl in Spellchecker.implementations:
+            libs.append(impl.getLibraryName())
+        return libs
```

Comment:
Humm.. more python way to do it : 
`return [i.getLibraryName() for i in Spellchecker.implementations]`